template_feed/Microsoft.DotNet.Common.ProjectTemplates.6.0/content/ClassLibrary-CSharp/.template.config/csharpFeatures.template.json

```diff
@@ -0,0 +1,113 @@
+﻿{
+  "$schema": "http://json.schemastore.org/template",
+  "symbols": {
+    "csharp10orlater": {
+      "type": "generated",
+      "generator": "regexMatch",
+      "datatype": "bool",
+      "parameters": {
+        "pattern": "10.0|10|preview|latest|default|latestMajor",
+        "source": "langVersion"
+      }
+    },
+    "csharp9orlater": {
+      "type": "generated",
+      "generator": "regexMatch",
+      "datatype": "bool",
+      "parameters": {
+        "pattern": "9|9.0|10.0|10|preview|latest|default|latestMajor",
+        "source": "langVersion"
+      }
+    },
+    "csharp8orlater": {
+      "type": "generated",
+      "generator": "regexMatch",
+      "datatype": "bool",
+      "parameters": {
+        "pattern": "8|8.0|9|9.0|10.0|10|preview|latest|default|latestMajor",
+        "source": "langVersion"
+      }
+    },
+    "csharpFeature_GlobalUsings_Supported": {
```

Comment:
Please rename to "csharpFeature_ImplicitUsings_supported"

Also consider whether "supported" is already implied and can be dropped. That's stylistic. 

Since there is a global using feature and an implicit using feature, the current name will be confusing (implicit usings relies on global usings but is separate).

Reply:
renamed it, removed `supported`